manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+PackageLocale: en-US
+Publisher: HoYoverse
+PublisherUrl: https://www.hoyoverse.com/en-us
+PackageName: Genshin Impact Launcher
+PackageUrl: https://genshin.hoyoverse.com/en/home
+License: copyright cognosphere
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate